saos-webapp/src/main/webapp/WEB-INF/view/details/judgmentDetailsNavigation.jsp

```diff
@@ -283,7 +283,7 @@
             <div class="legalBases">    
                 <c:forEach items="${judgment.legalBases}" var="legalBase" >
                     <div class="legalBase">
-                        <a href="${contextPath}/search?legalBase=${legalBase}" data-toggle="tooltip" title="${legalBasesLinkTooltip}" >
+                        <a href="${contextPath}/search?legalBase=%22${legalBase}%22" data-toggle="tooltip" title="${legalBasesLinkTooltip}" >
```

Comment:
I also added related task to escaping: #877 for situations where for example judgeName will contain quotes "
There will be needed some change in parsing queries